client/gutenberg/extensions/tiled-gallery/layout/index.js

```diff
@@ -0,0 +1,73 @@
+/**
+ * External dependencies
+ */
+import photon from 'photon';
+import { __, sprintf } from '@wordpress/i18n';
+import { Component } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import Square from './square';
+import GalleryImage from '../gallery-image';
+
+export default class Layout extends Component {
+	photonize( { height, width, url } ) {
+		const { layoutStyle } = this.props;
+		if ( [ 'circle', 'square' ].includes( layoutStyle ) && width && height ) {
+			const size = Math.min( width, height );
+			return photon( url, { resize: `${ size },${ size }` } );
+		}
+		return photon( url );
```

Comment:
Should we check for `window.location.hostname === 'localhost'` and return the original URL in that case?